### ISSUE TITLE
Disconnected if the receivers bags are full while the new reagent bags are not full

### ISSUE THREAD

**Issue by MrUzagi** [arkayenro/arkinventory#1695]
**NOTE**: This happens even with all add-ons disabled. But since AI is a bag add-on the issue might come up.

Both players in a trade gets disconnected if the receivers bags are full while having the new reagent bags that are not full.

What I've done to replicate it:
* Have add-ons on or off, doesn't matter.
* Get a reagent bag from the event vendor for the new bag slot
* Fill your regular bags
* The new reagent bag has some free slots
* Make a trade that will no go into the reagent bags
= Both players gets disconnected

This does not happen if the reagent bag also is full, instead you get the usual error message about full bags.

My guess is that someone at Blizzard forgot to update some bag sanity checks after adding the new reagent bag slot.

**Comment by arkayenro** [comment]
yeah, i cant fix that.  not sure i could even workaround it, not easily anyway.

**Comment by MrUzagi** [comment]
I wasn't expecting anyone other than Blizzard to fix it. I just posted it here as information as it concerns bags, and someone might *think* it's AI.
